exampleSite/content/awesome-team/yoda.md

```diff
@@ -1,15 +1,4 @@
 +++
-fragment = "member"
-#disabled = false
-date = "2017-10-17"
-lastmod = "2017-10-17"
-weight = 900
-#background = "dark"
-selfLink = "member-fragment"
-
-title = ""
-subtitle = ""
-
   [[member]]
```

Comment:
[[member]] is not necessary anymore. That would add another layer, which is unnecessary.